benchmarks/src/main/kotlin/com/google/samples/apps/nowinandroid/interests/InterestsActions.kt

```diff
@@ -35,7 +39,13 @@ fun MacrobenchmarkScope.goToInterestsScreen() {
 fun MacrobenchmarkScope.interestsScrollTopicsDownUp() {
     device.wait(Until.hasObject(By.res("interests:topics")), 5_000)
     val topicsList = device.findObject(By.res("interests:topics"))
-    device.flingElementDownUp(topicsList)
+    if (topicsList != null) {
+        // TODO: Ensure topics are availble.
+        device.flingElementDownUp(topicsList)
+    } else {
+        Log.e(TAG, "No topics found, can't scroll during baseline profile generation.")
+        fail()
```

Comment:
Would it make more sense to show the error in the `fail` function? You would see it in the test output.
If it's just in the Logcat, you would need to search for it in the logs and wouldn't see it in CI output for example.

```suggestion
        fail("No topics found, can't scroll during baseline profile generation.")
```